### PR TITLE
V1.11.0

### PR DESCRIPTION
**Pull request by amitnadirping**

### CODE REVIEW

File: sample/build.gradle [hunk 1]
```diff
@@ -54,52 +60,64 @@ dependencies {
...
+    implementation "androidx.credentials:credentials-play-services-auth:1.2.0"
+
+    /*
+     * OkHttp is a third-party library developed by Square for sending and receive HTTP-based
+     * network requests.
+     */
+    implementation 'com.squareup.okhttp3:okhttp:4.11.0'
     //Import the Firebase BoM (Bill of Materials)
-    implementation platform('com.google.firebase:firebase-bom:26.3.0')
+    implementation platform('com.google.firebase:firebase-bom:32.2.3')
```
Comment (EvgeniyMish) [inline sample/build.gradle:83]: this should be in release-notes

File: sample/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -64,6 +65,19 @@
         <activity android:name="com.pingidentity.pingone.TOTPActivity"/>
 
         <activity android:name=".ManualAuthActivity"/>
+        <activity
+            android:name=".NotificationTestsActivity"
+            android:excludeFromRecents="true"
+            android:exported="false" />
```
Comment (EvgeniyMish) [inline sample/src/main/AndroidManifest.xml]: this activity is not present in the current release ?

**Review by EvgeniyMish — CHANGES_REQUESTED** [review]
resolve the issues in comments

File: sample/build.gradle (hunk 1 above)
Comment (amitnadirping) [reply sample/build.gradle:83]: Done

File: sample/src/main/AndroidManifest.xml (hunk 2 above)
Comment (amitnadirping) [reply sample/src/main/AndroidManifest.xml]: Done

**Review by EvgeniyMish — APPROVED** [review]

**Review by adavid-pingidentity — APPROVED** [review]

**Pull request merged**